backend/pkg/api/handlers/handler_service.go

```diff
@@ -184,7 +193,7 @@ func (h *HandlerService) getCurrentChartTimeLimitsForDashboard(ctx context.Conte
 		return limits, err
 	}
 	limits.MinAllowedTs = limits.LatestExportedTs - min(maxAge, limits.LatestExportedTs)                        // min to prevent underflow
-	secondsPerEpoch := uint64(12 * 32)                                                                          // TODO: fetch dashboards chain id and use correct value for network once available
+	secondsPerEpoch := utils.Config.Chain.ClConfig.SlotsPerEpoch * utils.Config.Chain.ClConfig.SecondsPerSlot   // TODO: fetch dashboards chain id and use correct value for network once available
```

Comment:
Issue: I've been actively trying to avoid relying on global states in v2. If you insist on this change, pls add the config as a field of the `HandlerService` struct and pass it in the constructor in `api/router.go`.

Reply:
Well no I don't, but I don't think using magic constants is better either. With gnosis coming up this should already lead to wrong results I think.

I agree that this is also not a permanent solution (so I didn't touch the comment), but for now I'd say it's a better temporary one. I can apply your suggestion with the config param though